libtiny_tui/src/msg_area/line.rs

```diff
@@ -307,7 +270,7 @@ mod tests {
     fn height_test_1() {
         let mut line = Line::new();
         line.add_text("a b c d e");
-        assert_eq!(line.rendered_height(1), 5);
+        assert_eq!(line.rendered_height(1), 9);
```

Comment:
:+1: I thought maybe we should preserve the old behavior here, but then if I see something like

```
aaaaa|
aaaaa|
```

There's no way to know whether that's `aaaaaaaaaa` split into two lines or `aaaaa aaaaa`. If we preserve the space here then we render it as

```
aaaaa|
 aaaa|
a    |
```

Now it's clear that the original string is `aaaaa aaaaa`.